.github/workflows/publish-prisma-fmt-wasm.yml

```diff
@@ -0,0 +1,71 @@
+name: Build and publish @prisma/prisma-fmt-wasm
```

Comment:
This is the same as https://github.com/prisma/prisma-fmt-wasm/blob/main/.github/workflows/publish-prisma-fmt-wasm.yml but considerably simpler due to prisma-fmt-wasm being in the same repo as engines.